tests/test_stackrate.py

```diff
@@ -199,7 +200,7 @@ def test_stack_rate(self):
 
     def test_stackrate_error(self):
         """
-        Compare with legacy data
+        Compare with legacy data. Default behaviour is now 2-sigma, so mult legacy by 2.
```

Comment:
It's not clear to me exactly where "mult legacy by 2" is actually happening in the test code. Is this comment line still relevant?